docs/development_tips.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Tips and Tricks"
+---
+
+This page provides some tips and tricks that may help you as a contributor to Home Assistant. The list here is by no means exhaustive, and if you pick up any additional undocumented tips and tricks, please open a PR to add them here.
+
+## Tips and Tricks
+
+### Keep PRs simple
+
+It's in your best interest to keep your PRs as short and simple as possible. It may feel like extra work on your part to split a change you have in mind into smaller changes, but on the plus side:
+
+- Your code is less likely to get accepted with a bug because they will be easier to spot
```

Comment:
I think this section is talking about the positives of small PRs, so perhaps it needs to be inversed?

```suggestion
- Your code is more likely to be accepted because it will be much easier to review, and confirm it is bug free.
```

Reply:
great idea!